<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDiseaseWindow</class>
 <widget class="QDialog" name="PatientDiseaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutMain" stretch="1,11,1,3,1">
     <item>
      <widget class="QLabel" name="labelPatientHistoryWithSpinalDisease">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Анамнез пациента с заболеванием позвоночника&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-9</y>
          <width>960</width>
          <height>393</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutCenter" stretch="0,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,1,1,1,1,1,1,1">
             <item>
              <widget class="QLabel" name="labelDiseaseStartTime">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>С какого времени считает себя больным?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelOccurrenceOfDisease">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="accessibleName">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="accessibleDescription">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>С чем связывает возникновение болезни?</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="openExternalLinks">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelManifestationOfDisease">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Чем проявила себя болезнь?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRevealed">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Кем выявлена деформация?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelFirstWentToMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Когда впервые обратился в медицинское учреждение?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>В какое медицинское учреждение?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTreatmentTimeInDays">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Как долго проводилось лечение? (в днях)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTypeOfTreatment">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Вид лечения?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTreatmentResult">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Как оценивает результаты лечения?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelNewDisease">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Присоединились ли новые проявления болезни?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelWentToOtherMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Обращался ли повторно в другие мед. учреждения?</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1,1,1,1" columnstretch="1,1">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item row="9" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxNewDisease">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Да</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Нет</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Поликлиника</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Консультативный прием</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Стационар</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Прочее</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="10" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxWentToOtherMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Да</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Нет</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxOccurrenceOfDisease">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Не знает</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Травма</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Физическая нагрузка</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Переохлаждение</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Беременность</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Роды</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Прочее</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxRevealed">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Самостоятельно</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Родственник</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Окружающие</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Доктор</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Портниха</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Прочее</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxTreatmentResult">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Отлично</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Хорошо</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Удовлетворительно</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Плохо</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ухудшение</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QComboBox" name="comboBoxTypeOfTreatment">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Нет</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Консервативное</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Оперативное</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Санаторно - курортное</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QDateTimeEdit" name="dateTimeEditFirstWentToMedicalFacility">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QComboBox" name="comboBoxManifestationOfDisease">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Не проявляет</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Боли</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Деформация туловища</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Неврологические нарушения</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Прочее</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QComboBox" name="comboBoxTypeOfTreatmentAddition">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QLineEdit" name="lineEditTreatmentTimeInDays">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QDateTimeEdit" name="dateTimeEditDiseaseStart">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAddition_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дополнительные сведения:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditAddition"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutBottom" stretch="1,2,4">
       <item>
        <widget class="QPushButton" name="buttonClose">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonResult">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Результат</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelResult">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
